modules/datadict/php/fields.class.inc

```diff
@@ -23,8 +23,18 @@ class Fields extends \NDB_Page
      */
     public function handle(ServerRequestInterface $request) : ResponseInterface
     {
+        $user     = \NDB_Factory::singleton()->user();
+        $projects = $user->getProjectIDs();
+        $list_of_cohorts  = [];
+        foreach ($projects AS $projectID) {
+            $list_of_cohorts = $list_of_cohorts + \Utility::getCohortList($projectID);
+        }
+        $cohort_options=[];
+        foreach (array_values($list_of_cohorts) as $name) {
+            $cohort_options[$name] = $name;
+        }
```

Comment:
I do not like to name a variable `$list_of_x` as a plural variable name and its type already implies the variable contains a list (or other collection), I'd go back to `$cohorts` here.

```suggestion
        $cohorts  = [];
        foreach ($projects AS $projectID) {
            $cohorts = $cohorts + \Utility::getCohortList($projectID);
        }
        $cohort_options=[];
        foreach (array_values($cohorts) as $name) {
            $cohort_options[$name] = $name;
        }
```